virttest/env_process.py

```diff
@@ -575,6 +576,16 @@ def preprocess(test, params, env):
             except path.CmdNotFoundError, msg:
                 raise exceptions.TestSkipError(msg.message)
 
+    # enable network proxies setting in urllib2
+    if params.get("network_proxies"):
+        proxies = {}
+        for proxy in re.split("[\,\;]\s*", params["network_proxies"]):
+            proxy = dict([re.split("_proxy\:\s*", proxy)])
```

Comment:
the same here, I'm not familiar with a special meaning of the colon and please use the `r` prefix for regexps...